src/Products/DateRecurringIndex/tests.py

```diff
@@ -1,12 +1,22 @@
 # -*- coding: utf-8 -*-
-import unittest
 import doctest
+import unittest
+from datetime import datetime
+
+import pytz
```

Comment:
importing here will slow down whenever this module is imported - which is on test discovery, IIRC.
normally, pytz needs some time to be imported.
however, i'll remove this dependency in an upcoming PR, so it's fine to merge here!